src/plugin/applyViteHtmlTransform.js

```diff
@@ -14,7 +14,7 @@ const ssrRegex = RegExp(toSSRComment('([0-9]+)'), 'g')
  * @property {import('./componentAttrStore').ComponentAttrStore} componentAttrStore
  * @param {ApplyViteHtmlTransformParams}
  * @param {import('.').SlinkityConfigOptions}
- * @returns {string} - HTML with statically rendered content and Vite transforms applied
+ * @returns {Promise<string>} - HTML with statically rendered content and Vite transforms applied
```

Comment:
Ah, thanks for fixing that type 👍 Could really use an eslint plugin to flag bad TS docs...

Reply:
https://www.npmjs.com/package/eslint-plugin-jsdoc might be what you're looking for